packages/eas-cli/src/build/ios/prepareJob.ts

```diff
@@ -18,10 +13,7 @@ import { Platform } from '../types';
 interface JobData {
   projectArchive: ArchiveSource;
   credentials?: IosCredentials;
-  projectConfiguration: {
-    iosBuildScheme?: string;
-    iosApplicationTarget?: string;
-  };
+  buildScheme: string;
```

Comment:
scheme should be accessed from the build profile when creating job object

Reply:
discussed offline